### PR TITLE
Implement a filter to remove troublesome individuals from the game.

### PR DESCRIPTION
**Pull request by seanmtracey**

### CODE REVIEW

File: bin/lib/barnier-filter.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const debug = require('debug')('bin:lib:barner-filter');
+
+const CANDIDATES = process.env.BARNIER_LIST || [];
```
Comment (Lily2point0) [inline bin/lib/barnier-filter.js]: Should the list be a JS/JSON file to be more manageable?

File: bin/lib/game.js [hunk 2]
```diff
@@ -36,8 +37,10 @@ class Game{
 	selectRandomSeedPerson(){
 		return correlations_service.allIslands()
 			.then(islands => {
-				const biggestIsland = islands[0];
-				const mostConnectedIndividuals = Object.keys(biggestIsland).map(person => {
+				const biggestIsland = Object.keys(islands[0]).filter(person => {
+					return !barnierFilter(person.replace('people:', ''));
```
Comment (Lily2point0) [inline bin/lib/game.js]: Wouldn't that work just for the seed person, but not the possible answers?

File: bin/lib/barnier-filter.js (hunk 1 above)
Comment (seanmtracey) [inline bin/lib/barnier-filter.js]: I don't think we should hard code values into our apps. We can parse the env var as JSON if you think that would be better/more readable?

File: bin/lib/game.js (hunk 2 above)
Comment (seanmtracey) [inline bin/lib/game.js]: Correct. Will ammend.

**Pull request merged**